content/en/docs/components/training/legacy-v1/installation.md

```diff
@@ -0,0 +1,99 @@
+The Training Operator is a lightweight Kubernetes controller that orchestrates the
+appropriate Kubernetes workloads to perform distributed ML training and fine-tuning.
+
+## Prerequisites
+
+These are the minimal requirements to install the Training Operator:
+
+- Kubernetes >= 1.28
+- `kubectl` >= 1.28
+- Python >= 3.7
+
+## Installing the Training Operator
+
+You need to install the Training Operator control plane and Python SDK to create training jobs.
+
+### Installing the Control Plane
+
+You can skip these steps if you have already
+[installed Kubeflow platform](https://www.kubeflow.org/docs/started/installing-kubeflow/)
+using manifests or package distributions. The Kubeflow platform includes the Training Operator.
+
+You can install the Training Operator as a standalone component.
+
+Run the following command to install the stable release of the Training Operator control plane: `v1.8.1`
+
+```shell
+kubectl apply --server-side -k "github.com/kubeflow/training-operator.git/manifests/overlays/standalone?ref=v1.8.1"
+```
+
+Run the following command to install the latest changes of Training Operator control plane:
+
+```shell
+kubectl apply --server-side -k "github.com/kubeflow/training-operator.git/manifests/overlays/standalone?ref=master"
+```
+
+After installing it, you can verify that Training Operator controller is running as follows:
+
+```shell
+$ kubectl get pods -n kubeflow
+
+NAME                                             READY   STATUS    RESTARTS   AGE
+training-operator-658c68d697-46zmn               1/1     Running   0          90s
+```
+
+Run this command to check installed Kubernetes CRDs for each supported ML framework:
+
+```shell
+$ kubectl get crd
+
+mpijobs.kubeflow.org                                     2023-06-09T00:31:07Z
+mxjobs.kubeflow.org                                      2023-06-09T00:31:05Z
+paddlejobs.kubeflow.org                                  2023-06-09T00:31:09Z
+pytorchjobs.kubeflow.org                                 2023-06-09T00:31:06Z
+tfjobs.kubeflow.org                                      2023-06-09T00:31:04Z
+xgboostjobs.kubeflow.org                                 2023-06-09T00:31:04Z
+```
+
+### Installing the Python SDK
+
+The Training Operator [implements a Python SDK](https://pypi.org/project/kubeflow-training/)
+to simplify creation of distributed training and fine-tuning jobs for Data Scientists.
+
+Run the following command to install the latest stable release of the Training SDK:
+
+```shell
+pip install -U kubeflow-training
+```
+
+Run the following command to install the latest changes of Training SDK:
+
+```shell
+pip install git+https://github.com/kubeflow/training-operator.git@master#subdirectory=sdk/python
+```
+
+Otherwise, you can also install the Training SDK using the specific GitHub commit, for example:
+
+```shell
+pip install git+https://github.com/kubeflow/training-operator.git@7345e33b333ba5084127efe027774dd7bed8f6e6#subdirectory=sdk/python
```

Comment:
What about a release version that is a little more pinned than a branch, but not as fine grained as needing a specific  commit?

Reply:
They can select it from PyPi directly:
```
pip install kubeflow-training==1.9.0rc0
```